Makefile

```diff
@@ -15,6 +24,8 @@ ifdef IN_VENV
 else
 	$(POETRY) install --remove-untracked
 endif
+	@poetry run ipython profile create --ipython-dir=build/ipython
```

Comment:
Is creating the profile idempotent? As in if it exists already this won't fail the second time we run it?

Reply:
We can invoke it multiple times, it won't fail